### PR TITLE
Refactored security context tests

### PR DESCRIPTION
**Pull request by csgregorian**
Refactored security context tests to replace numbered errors with constants.

### CODE REVIEW

File: cmd/securityContext.go [hunk 1]
```diff
@@ -12,39 +12,37 @@ func checkSecurityContext(container apiv1.Container, result *Result) {
...
 	}
-
-	return
 }
 
 func printResultSC(results []Result) {
 	for _, result := range results {
 		switch err := result.err; err {
-		case 1:
+		case ErrorSecurityContextNIL:
```
Comment (klautcomputing) [inline cmd/securityContext.go]: Can you standardize the logging to be in line with what we do in here: https://github.com/Shopify/kubeaudit/blob/master/cmd/image.go#L33

File: cmd/errors.go [hunk 2]
```diff
@@ -9,4 +9,6 @@ const (
 	ErrorReadOnlyRootFilesystemFalse
 	ErrorRunAsNonRootNIL
 	ErrorRunAsNonRootFalse
+	ErrorCapabilitiesNIL
+	ErrorCapabilitiesAddedOrNotDropped
```
Comment (jinankjain) [inline cmd/errors.go]: Not sure about this that we want to merge this two error condition or not? I remember having a discussion with @klautcomputing

Comment (klautcomputing) [reply cmd/errors.go]: @csgregorian and I discussed that it's easier for now to just have one, get all the errors named in kubeaudit and then once this is done we will search for a way to support multiple errors in one result. That change could go hand in hand with the log level change and we could come up with a something that supports both multiple errors and different log levels.
But let's see about that.
I am fine to merge this as is and address it later.

Comment (jinankjain) [reply cmd/errors.go]: cool

**Pull request merged**